src/test/java/com/salesforce/storm/spout/sideline/handler/SidelineSpoutHandlerTest.java

```diff
@@ -449,7 +397,178 @@ public void testGenerateVirtualSpoutId() {
 
         final SidelineVirtualSpoutIdentifier sidelineVirtualSpoutIdentifier = (SidelineVirtualSpoutIdentifier) virtualSpoutIdentifier;
 
-        assertEquals(expectedPrefix, sidelineVirtualSpoutIdentifier.getConsumerId());
+        assertEquals(CONSUMER_ID_PREFIX, sidelineVirtualSpoutIdentifier.getConsumerId());
         assertEquals(expectedSidelineRequestIdentifier, sidelineVirtualSpoutIdentifier.getSidelineRequestIdentifier());
     }
+
+    /**
+     * Test that periodically calling {@link SidelineSpoutHandler#loadSidelines()} will restore {@link VirtualSpout}'s and
+     * {@link com.salesforce.storm.spout.dynamic.filter.FilterChainStep}'s after calling
+     * {@link SidelineSpoutHandler#onSpoutOpen(DynamicSpout, Map, TopologyContext)}, and that when spouts and filters are removed
+     * they are properly restored back onto the spout.
+     */
+    @Test
```

Comment:
So this tests if the vspouts are no longer there, but what if you call loadSidelines and they're still there?

Reply:
Good call, I added another call to loadSidelines() and another set of assertions to verify nothing has changed.